frontend/packages/console-shared/src/hooks/scroll.ts

```diff
@@ -1,4 +1,5 @@
 import * as React from 'react';
+import { useResizeObserver, useDebounceCallback } from '@console/shared';
```

Comment:
use relative path to import these
```suggestion
import { useResizeObserver } from './useResizeObserver';
import { useDebounceCallback } from './debounce';
```

Reply:
updated